epistasis_selection.py

```diff
@@ -49,10 +50,25 @@ def combine_k_rounds(num_rounds: int, mutations: Iterable[Tuple[Tuple[int, int],
   return all_samples
 
 
+def filter_mutation_set_by_position(mutation_sets, limit: int = 10):
+  """filter the set of mutations so that they're only used a maximum of `limit` times"""
+  filtered_mutation_sets = []
+  position_counter = Counter()
+  for mutation_set in mutation_sets:
+    positions = [m[0] for m in mutation_set]
+    if any([position_counter[position] >= limit for position in positions]):
+      continue
+    else:
+      position_counter.update(positions)
+      filtered_mutation_sets.append(mutation_set)
+  return filtered_mutation_sets
+
+
 def get_epistatic_seqs_for_landscape(landscape: potts_model.PottsModel,
                                      distance: int,
                                      n: int,
                                      adaptive: bool = True,
+                                     max_reuse: Optional[int] = None,
```

Comment:
update the docstring to include this. you should be running a linter before you submit. it would catch things like this.